update hooks docs (#112)
MadeInHaus/haus-core: apps/web/pages/hooks/use-isomorphic-layout-effect.mdx

## Import

```tsx copy
import { useIsomorphicLayoutEffect } from '@madeinhaus/hooks';
```

## Parameters

> - `effect` _(EffectCallback)_: A function that will be called after the component has rendered.

Reviewer: ```suggestion
- `effect` _(EffectCallback)_: A function that will be called before the browser repaints the screen.
```

- `deps` _(DependencyList, optional)_: An array of dependencies that will trigger the effect when they change. If not specified, the effect will run after every render.

## Returns
- _(void)_: This hook does not return anything.

## Usage

```tsx copy
import { useIsomorphicLayoutEffect } from '@madeinhaus/hooks';

function MyComponent() {
  useIsomorphicLayoutEffect(() => {
    console.log('Component has rendered');
  });

  return (
    <div>Hello, world!</div>
  );
}
```

> In this example, we import the `useIsomorphicLayoutEffect` hook from the `@madeinhaus/hooks` package and use it to log a message after the component has rendered.

Reviewer: ```suggestion
In this example, we import the `useIsomorphicLayoutEffect` hook from the `@madeinhaus/hooks` package and use it to log a message before the browser repaints the screen.
```
